app/views/layouts/application.html.erb

```diff
@@ -28,7 +28,7 @@
     <header class="govuk-header" role="banner" data-module="govuk-header">
       <div class="govuk-header__container govuk-width-container">
         <div class="govuk-header__logo">
-          <%= link_to "/", class: "govuk-header__link govuk-header__link--homepage" do %>
+          <%= govuk_link_to "/", class: "govuk-header__link govuk-header__link--homepage" do %>
             <span class="govuk-header__logotype">
```

Comment:
Could you update this svg one as well

Reply:
I'd rather not - it looks complicated (fallback images, what is this?), and more importantly, css for the classes that live on `application` svgs aren't in our code - they are from `govuk-frontend`. So we would have to add our versions of them to our code, with small differences, and then worry about maintaining them.

Also, I am quite happy keeping `application` file complicated and unreadable 😂

Reply:
👍